language_detector/main.py

```diff
@@ -1,8 +1,65 @@
 # -*- coding: utf-8 -*-
 
 """This is the entry point of the program."""
+import re
+import collections
 
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    # counter var created for each language, initiated with '0'
+    dictionar_of_languages = {}
+    for lang in languages:
+        dictionar_of_languages[lang['name']] = 0
+
+    # finds all words from string and strips the unnecessary symbols
+    list_of_words_in_text = re.findall(r'\w+',text)
```

Comment:
Used `findall` to do what split does and also remove the non-alphanumeric symbols at the same time. Thanks, for suggestion, I'll look into it.